molecule/default/prepare.yml

```
@@ -1,10 +1,15 @@
 ---
-# boto3 is installed anywhere we would be applying this Ansible role
-- name: Prepare
+- name: Import upgrade playbook
+  ansible.builtin.import_playbook: upgrade.yml
+
+- name: Import python playbook
+  ansible.builtin.import_playbook: python.yml
+
+# boto3 is pre-installed anywhere we would be applying this Ansible
+# role
+- name: Install boto3 and pip
```

Since `pip` is installed in the `python.yml` playbook above.

```suggestion
- name: Install boto3
```